library/jruby

```diff
@@ -13,15 +13,15 @@ latest: git://github.com/cpuguy83/docker-jruby@882f6121c19e90e158bc86645335f8762
 1.7-onbuild: git://github.com/cpuguy83/docker-jruby@882f6121c19e90e158bc86645335f8762d26ff19 1.7/onbuild
 1.7.20-onbuild: git://github.com/cpuguy83/docker-jruby@882f6121c19e90e158bc86645335f8762d26ff19 1.7/onbuild
 
-9.0.0.0: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0.pre2: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
-9.0.0.0.pre2-jre: git://github.com/cpuguy83/docker-jruby@0f5ca4efd34cedd56e7e2c7138f98c5e80402c6b 9000/jre
+9.0.0.0: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0-jre: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0.rc1: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
+9.0.0.0.rc1-jre: git://github.com/cpuguy83/docker-jruby@14113750316f06b2785a41b5bc3dd6b23510af6f 9000/jre
```

Comment:
Not sure where this commit ID comes from, but I think you meant d7f7b44668aeb8f192d210302aff23bfde1ab7fa (https://github.com/cpuguy83/docker-jruby/commit/d7f7b44668aeb8f192d210302aff23bfde1ab7fa).